plugins/plugin-docker/che-plugin-openshift-client/src/main/java/org/eclipse/che/plugin/openshift/client/OpenShiftConnector.java

```diff
@@ -136,6 +142,7 @@
     private final String          openShiftCheServiceAccount;
     private final int             openShiftLivenessProbeDelay;
     private final int             openShiftLivenessProbeTimeout;
+    private final String          persistentVolumeClaim;
```

Comment:
I would rather call it `workspacesPersistentVolumeClaim` or `workspacesPVC` to make it clear that it will be shared by all workpaces.